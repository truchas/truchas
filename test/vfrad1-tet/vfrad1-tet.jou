## /opt/cubit/11.1/bin/clarox
## Cubit Version 11.1
## Cubit Build 39271
## Revised 2009-02-17 11:06:39 -0700 (Tue, 17 Feb 2009)
## Running 05/04/2010 01:31:56 PM
## Command Options:
## -warning = On
## -information = On

create torus major radius 1 minor radius 0.8
webcut volume 1 with plane yplane offset 0 noimprint nomerge
webcut volume 2 with plane yplane offset 0 rotate 60 about z noimprint nomerge
delete volume 1 3
webcut volume 2 with plane zplane offset 0 rotate 60 about z noimprint nomerge
delete volume 4
create torus major radius 0.9 minor radius 0.4
webcut volume 2 tool volume 5
delete volume 5
imprint volume all
merge volume all

curve 11 12 interval 32
curve 11 scheme bias factor 1.04 start vertex 5
curve 12 scheme bias factor 1.04 start vertex 4
mesh curve 11 12

curve 20 22 interval 32
curve 20 scheme bias factor 1.03 start vertex 13
curve 22 scheme bias factor 1.03 start vertex 14
mesh curve 20 22

curve 26 25 23 24 interval 6
curve 26 25 23 24 scheme equal

surface 22 25 scheme map
mesh surface 22 25

curve 8 19 21 10 interval 8
curve 8 19 21 10 scheme equal
mesh curve 8 19 21 10

volume 2 scheme sweep source surface 25 target surface 22 rotate off
volume 2 sweep smooth linear
mesh volume 2

surface 27 scheme pave
mesh surface 27
surface 27 smooth scheme winslow
smooth surface 27

volume 6 scheme sweep source surface 27 target surface 29 rotate off
volume 6 sweep smooth linear
mesh volume 6

htet volume 2 6 unstructured

block 1 volume 2
block 2 volume 6

sideset 1 surface 25 27 23 28 24 22 29
sideset 2 surface 15
sideset 3 surface 21 wrt volume 2
sideset 5 surface 21

set large exodus file off
export genesis "vfrad1-tet.gen" overwrite
