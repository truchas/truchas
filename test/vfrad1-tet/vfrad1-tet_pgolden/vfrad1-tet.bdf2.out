
TRAP step      1: T= 0.00000E+00, H= 1.00000E-03, ETAH= 5.00000E-04
  Preconditioner updated at T= 1.00000E-03, ETAH= 5.00000E-04
    1: error= 9.19204E+00
    2: error= 5.60829E+00
    3: error= 4.73632E-01
    4: error= 9.50155E-03
  AIN BCE solve succeeded:   4 iterations, error= 9.50155E-03
  Step accepted: no local error control

BDF2 step      2: T= 1.00000E-03, H= 1.00000E-03, ETAH= 6.66667E-04
    1: error= 1.18754E+01
    2: error= 7.23918E+00
    3: error= 1.13521E+00
    4: error= 3.33961E-02
  AIN BCE solve succeeded:   4 iterations, error= 3.33961E-02
  Step accepted: no local error control

BDF2 step      3: T= 2.00000E-03, H= 1.00000E-03, ETAH= 6.66667E-04
    1: error= 4.12843E+00
    2: error= 2.65120E+00
    3: error= 4.19166E-01
    4: error= 1.04986E-02
  AIN BCE solve succeeded:   4 iterations, error= 1.04986E-02
  Step accepted: no local error control

BDF2 step      4: T= 3.00000E-03, H= 1.00000E-03, ETAH= 6.66667E-04
    1: error= 1.38259E+00
    2: error= 6.50980E-01
    3: error= 1.76990E-01
    4: error= 3.05643E-03
  AIN BCE solve succeeded:   4 iterations, error= 3.05643E-03
  Step accepted: perr= 1.62958E+00

BDF2 step      5: T= 4.00000E-03, H= 4.94054E-04, ETAH= 3.71279E-04
    1: error= 2.38157E-01
    2: error= 4.59875E-02
  AIN BCE solve succeeded:   2 iterations, error= 4.59875E-02
  Step accepted: perr= 2.10614E-01

BDF2 step      6: T= 4.49405E-03, H= 1.07937E-03, ETAH= 6.40196E-04
    1: error= 2.88637E-01
    2: error= 6.46623E-02
    3: error= 6.94098E-03
  AIN BCE solve succeeded:   3 iterations, error= 6.94098E-03
  Step accepted: perr= 3.46366E-01

BDF2 step      7: T= 5.57343E-03, H= 1.09100E-03, ETAH= 7.26037E-04
    1: error= 1.46025E-01
    2: error= 3.16604E-02
  AIN BCE solve succeeded:   2 iterations, error= 3.16604E-02
  Step accepted: perr= 1.77687E-01

BDF2 step      8: T= 6.66443E-03, H= 1.67211E-03, ETAH= 1.04171E-03
    1: error= 2.06396E-01
    2: error= 1.04435E-01
    3: error= 6.51129E-02
    4: error= 6.56113E-03
  AIN BCE solve succeeded:   4 iterations, error= 6.56113E-03
  Step accepted: perr= 3.71272E-01

BDF2 step      9: T= 8.33654E-03, H= 1.63969E-03, ETAH= 1.09670E-03
    1: error= 1.50871E-01
    2: error= 5.88803E-02
    3: error= 6.78005E-02
    4: error= 1.42426E-02
  AIN BCE solve succeeded:   4 iterations, error= 1.42426E-02
  Step accepted: perr= 2.91808E-01

BDF2 step     10: T= 9.97623E-03, H= 2.06067E-03, ETAH= 1.32359E-03
    1: error= 1.82075E-01
    2: error= 9.08738E-02
    3: error= 9.68126E-02
    4: error= 4.45717E-02
  AIN BCE solve succeeded:   4 iterations, error= 4.45717E-02
  Step accepted: perr= 4.14363E-01

BDF2 step     11: T= 1.20369E-02, H= 2.07211E-03, ETAH= 1.38013E-03
    1: error= 1.47671E-01
    2: error= 7.05184E-02
    3: error= 7.22633E-02
    4: error= 5.15558E-02
    5: error= 2.47581E-03
  AIN BCE solve succeeded:   5 iterations, error= 2.47581E-03
  Step accepted: perr= 3.36376E-01

BDF2 step     12: T= 1.41090E-02, H= 2.45958E-03, ETAH= 1.59428E-03
    1: error= 1.88337E-01
    2: error= 7.24330E-02
    3: error= 7.57115E-02
    4: error= 7.00015E-02
    5: error= 1.43062E-02
  AIN BCE solve succeeded:   5 iterations, error= 1.43062E-02
  Step accepted: perr= 3.65522E-01

BDF2 step     13: T= 1.65686E-02, H= 2.69672E-03, ETAH= 1.77067E-03
    1: error= 2.11978E-01
    2: error= 7.37362E-02
    3: error= 7.44936E-02
    4: error= 6.42325E-02
    5: error= 4.89428E-02
  AIN BCE solve succeeded:   5 iterations, error= 4.89428E-02
  Step accepted: perr= 3.53760E-01

BDF2 step     14: T= 1.92653E-02, H= 3.03016E-03, ETAH= 1.98165E-03
    1: error= 2.46697E-01
    2: error= 8.60180E-02
    3: error= 8.62277E-02
    4: error= 6.31475E-02
    5: error= 9.88569E-02
    6: error= 3.70754E-03
  AIN BCE solve succeeded:   6 iterations, error= 3.70754E-03
  Step accepted: perr= 4.28263E-01

BDF2 step     15: T= 2.22955E-02, H= 3.08120E-03, ETAH= 2.04843E-03
  Preconditioner updated at T= 2.53767E-02, ETAH= 2.04843E-03
    1: error= 4.16905E-01
    2: error= 6.95097E-02
    3: error= 3.55934E-03
  AIN BCE solve succeeded:   3 iterations, error= 3.55934E-03
  Step accepted: perr= 4.16819E-01

BDF2 step     16: T= 2.53767E-02, H= 3.30689E-03, ETAH= 2.17893E-03
    1: error= 4.22167E-01
    2: error= 7.93671E-02
    3: error= 7.79856E-03
  AIN BCE solve succeeded:   3 iterations, error= 7.79856E-03
  Step accepted: perr= 4.44488E-01

BDF2 step     17: T= 2.86836E-02, H= 3.40496E-03, ETAH= 2.25897E-03
    1: error= 3.82354E-01
    2: error= 8.03188E-02
    3: error= 1.20432E-02
  AIN BCE solve succeeded:   3 iterations, error= 1.20432E-02
  Step accepted: perr= 4.14153E-01

BDF2 step     18: T= 3.20885E-02, H= 3.67459E-03, ETAH= 2.41902E-03
    1: error= 3.72176E-01
    2: error= 8.79987E-02
    3: error= 2.18422E-02
  AIN BCE solve succeeded:   3 iterations, error= 2.18422E-02
  Step accepted: perr= 4.28880E-01

BDF2 step     19: T= 3.57631E-02, H= 3.83940E-03, ETAH= 2.54102E-03
    1: error= 3.50803E-01
    2: error= 9.10517E-02
    3: error= 2.79965E-02
  AIN BCE solve succeeded:   3 iterations, error= 2.79965E-02
  Step accepted: perr= 4.19960E-01

BDF2 step     20: T= 3.96025E-02, H= 4.08247E-03, ETAH= 2.69409E-03
    1: error= 3.37057E-01
    2: error= 9.62934E-02
    3: error= 3.65396E-02
  AIN BCE solve succeeded:   3 iterations, error= 3.65396E-02
  Step accepted: perr= 4.22165E-01

BDF2 step     21: T= 4.36850E-02, H= 4.31908E-03, ETAH= 2.85261E-03
    1: error= 3.19073E-01
    2: error= 9.98210E-02
    3: error= 4.62256E-02
  AIN BCE solve succeeded:   3 iterations, error= 4.62256E-02
  Step accepted: perr= 4.19344E-01

BDF2 step     22: T= 4.80041E-02, H= 4.58242E-03, ETAH= 3.02511E-03
    1: error= 2.98929E-01
    2: error= 9.97981E-02
    3: error= 5.79888E-02
    4: error= 1.28849E-02
  AIN BCE solve succeeded:   4 iterations, error= 1.28849E-02
  Step accepted: perr= 4.29493E-01

BDF2 step     23: T= 5.25865E-02, H= 4.79796E-03, ETAH= 3.17432E-03
    1: error= 2.67773E-01
    2: error= 8.46888E-02
    3: error= 5.29074E-02
    4: error= 1.52278E-02
  AIN BCE solve succeeded:   4 iterations, error= 1.52278E-02
  Step accepted: perr= 3.78613E-01

BDF2 step     24: T= 5.73844E-02, H= 5.40106E-03, ETAH= 3.53110E-03
    1: error= 3.02305E-01
    2: error= 9.56037E-02
    3: error= 8.34107E-02
    4: error= 2.25945E-02
  AIN BCE solve succeeded:   4 iterations, error= 2.25945E-02
  Step accepted: perr= 4.33192E-01

BDF2 step     25: T= 6.27855E-02, H= 5.49549E-03, ETAH= 3.65311E-03
    1: error= 2.92264E-01
    2: error= 7.97054E-02
    3: error= 7.26012E-02
    4: error= 2.90320E-02
  AIN BCE solve succeeded:   4 iterations, error= 2.90320E-02
  Step accepted: perr= 3.89376E-01

BDF2 step     26: T= 6.82810E-02, H= 6.11241E-03, ETAH= 4.00401E-03
    1: error= 2.90492E-01
    2: error= 9.93120E-02
    3: error= 8.13100E-02
    4: error= 3.98571E-02
  AIN BCE solve succeeded:   4 iterations, error= 3.98571E-02
  Step accepted: perr= 4.23009E-01

BDF2 step     27: T= 7.43934E-02, H= 6.36283E-03, ETAH= 4.21369E-03
    1: error= 2.71438E-01
    2: error= 1.27738E-01
    3: error= 9.97007E-02
    4: error= 4.85936E-02
  AIN BCE solve succeeded:   4 iterations, error= 4.85936E-02
  Step accepted: perr= 3.99869E-01

BDF2 step     28: T= 8.07562E-02, H= 6.92192E-03, ETAH= 4.55077E-03
    1: error= 2.93162E-01
    2: error= 1.61392E-01
    3: error= 1.20052E-01
    4: error= 5.95964E-02
    5: error= 7.18066E-03
  AIN BCE solve succeeded:   5 iterations, error= 7.18066E-03
  Step accepted: perr= 4.29152E-01

BDF2 step     29: T= 8.76782E-02, H= 7.19128E-03, ETAH= 4.76388E-03
    1: error= 2.67793E-01
    2: error= 1.40667E-01
    3: error= 1.22334E-01
    4: error= 5.06173E-02
    5: error= 9.29034E-03
  AIN BCE solve succeeded:   5 iterations, error= 9.29034E-03
  Step accepted: perr= 4.06356E-01

BDF2 step     30: T= 9.48694E-02, H= 7.79141E-03, ETAH= 5.12583E-03
    1: error= 3.03161E-01
    2: error= 2.06779E-01
    3: error= 1.55714E-01
    4: error= 7.32988E-02
    5: error= 1.31667E-02
  AIN BCE solve succeeded:   5 iterations, error= 1.31667E-02
  Step accepted: perr= 4.23912E-01

BDF2 step     31: T= 1.02661E-01, H= 8.16537E-03, ETAH= 5.40138E-03
    1: error= 2.84287E-01
    2: error= 1.92717E-01
    3: error= 1.44515E-01
    4: error= 6.70760E-02
    5: error= 1.55505E-02
  AIN BCE solve succeeded:   5 iterations, error= 1.55505E-02
  Step accepted: perr= 4.30103E-01

BDF2 step     32: T= 1.10826E-01, H= 8.55491E-03, ETAH= 5.65933E-03
    1: error= 2.81464E-01
    2: error= 1.97529E-01
    3: error= 1.48996E-01
    4: error= 6.71467E-02
    5: error= 1.69830E-02
  AIN BCE solve succeeded:   5 iterations, error= 1.69830E-02
  Step accepted: perr= 4.36028E-01

BDF2 step     33: T= 1.19381E-01, H= 8.94861E-03, ETAH= 5.92134E-03
    1: error= 2.71180E-01
    2: error= 1.80417E-01
    3: error= 1.24630E-01
    4: error= 5.21407E-02
    5: error= 1.73556E-02
  AIN BCE solve succeeded:   5 iterations, error= 1.73556E-02
  Step accepted: perr= 4.34508E-01

BDF2 step     34: T= 1.28330E-01, H= 9.38513E-03, ETAH= 6.20748E-03
    1: error= 2.96936E-01
    2: error= 2.28592E-01
    3: error= 1.16525E-01
    4: error= 4.69704E-02
  AIN BCE solve succeeded:   4 iterations, error= 4.69704E-02
  Step accepted: perr= 4.14021E-01

BDF2 step     35: T= 1.37715E-01, H= 1.00919E-02, ETAH= 6.64754E-03
    1: error= 4.37005E-01
    2: error= 4.70206E-01
    3: error= 2.90179E-01
    4: error= 4.46896E-02
  AIN BCE solve succeeded:   4 iterations, error= 4.46896E-02
  Step accepted: perr= 4.59108E-01

BDF2 step     36: T= 1.47807E-01, H= 1.01482E-02, ETAH= 6.75919E-03
    1: error= 3.89170E-01
    2: error= 3.92900E-01
    3: error= 2.74958E-01
    4: error= 8.38725E-02
    5: error= 2.88083E-02
  AIN BCE solve succeeded:   5 iterations, error= 2.88083E-02
  Step accepted: perr= 4.56212E-01

BDF2 step     37: T= 1.57955E-01, H= 1.05038E-02, ETAH= 6.96256E-03
    1: error= 4.72890E-01
    2: error= 4.73826E-01
    3: error= 2.53571E-01
    4: error= 6.15750E-02
    5: error= 2.98869E-02
  AIN BCE solve succeeded:   5 iterations, error= 2.98869E-02
  Step accepted: perr= 3.96278E-01

BDF2 step     38: T= 1.68459E-01, H= 1.17075E-02, ETAH= 7.66649E-03
    1: error= 5.13236E-01
    2: error= 6.54550E-01
    3: error= 4.37372E-01
    4: error= 7.50743E-02
    5: error= 4.06700E-02
  AIN BCE solve succeeded:   5 iterations, error= 4.06700E-02
  Step accepted: perr= 4.05086E-01

BDF2 step     39: T= 1.80166E-01, H= 1.24255E-02, ETAH= 8.20230E-03
  Preconditioner updated at T= 1.92592E-01, ETAH= 8.20230E-03
    1: error= 5.91062E-01
    2: error= 1.17976E-01
    3: error= 5.21534E-02
    4: error= 2.03480E-02
  AIN BCE solve succeeded:   4 iterations, error= 2.03480E-02
  Step accepted: perr= 5.92389E-01

BDF2 step     40: T= 1.92592E-01, H= 1.08028E-02, ETAH= 7.37357E-03
    1: error= 2.42640E-01
    2: error= 6.54064E-02
    3: error= 4.05117E-02
  AIN BCE solve succeeded:   3 iterations, error= 4.05117E-02
  Step accepted: perr= 2.81652E-01

BDF2 step     41: T= 2.03394E-01, H= 1.53876E-02, ETAH= 9.69281E-03
    1: error= 3.99602E-01
    2: error= 1.46332E-01
    3: error= 7.64974E-02
    4: error= 5.87391E-02
    5: error= 1.49073E-01
    6: error= 9.90441E-02
    7: error= 4.28005E-03
  AIN BCE solve succeeded:   7 iterations, error= 4.28005E-03
  Step accepted: perr= 7.44544E-01

BDF2 step     42: T= 2.18782E-01, H= 1.07934E-02, ETAH= 7.64267E-03
    1: error= 2.50347E-01
    2: error= 4.41497E-02
  AIN BCE solve succeeded:   2 iterations, error= 4.41497E-02
  Step accepted: perr= 2.30508E-01

BDF2 step     43: T= 2.29576E-01, H= 1.79113E-02, ETAH= 1.10292E-02
    1: error= 4.48380E-01
    2: error= 1.62197E-01
    3: error= 6.88687E-02
    4: error= 2.62348E-02
  AIN BCE solve succeeded:   4 iterations, error= 2.62348E-02
  Step accepted: perr= 5.81094E-01

BDF2 step     44: T= 2.47487E-01, H= 1.41764E-02, ETAH= 9.83239E-03
    1: error= 2.62275E-01
    2: error= 8.66179E-02
    3: error= 4.96210E-02
  AIN BCE solve succeeded:   3 iterations, error= 4.96210E-02
  Step accepted: perr= 3.29559E-01

BDF2 step     45: T= 2.61663E-01, H= 1.81879E-02, ETAH= 1.16442E-02
    1: error= 3.40686E-01
    2: error= 1.95722E-01
    3: error= 1.19708E-01
    4: error= 1.33231E-01
    5: error= 2.51784E-01
    6: error= 2.85807E-01
    7: error= 1.61707E-02
  AIN BCE solve succeeded:   7 iterations, error= 1.61707E-02
  Step accepted: perr= 1.18431E+00

BDF2 step     46: T= 2.79851E-01, H= 1.02874E-02, ETAH= 7.55717E-03
    1: error= 2.07746E-01
    2: error= 3.44064E-02
  AIN BCE solve succeeded:   2 iterations, error= 3.44064E-02
  Step accepted: perr= 1.89007E-01

BDF2 step     47: T= 2.90138E-01, H= 2.11672E-02, ETAH= 1.26527E-02
    1: error= 3.94937E-01
    2: error= 1.43389E-01
    3: error= 5.55277E-02
    4: error= 2.26223E-02
  AIN BCE solve succeeded:   4 iterations, error= 2.26223E-02
  Step accepted: perr= 5.56938E-01

BDF2 step     48: T= 3.11306E-01, H= 1.64592E-02, ETAH= 1.14504E-02
    1: error= 2.02407E-01
    2: error= 8.98058E-02
    3: error= 5.47509E-02
    4: error= 4.08987E-02
  AIN BCE solve succeeded:   4 iterations, error= 4.08987E-02
  Step accepted: perr= 3.02215E-01

BDF2 step     49: T= 3.27765E-01, H= 2.16925E-02, ETAH= 1.38294E-02
    1: error= 3.16535E-01
    2: error= 1.85802E-01
    3: error= 1.32209E-01
    4: error= 1.87644E-01
    5: error= 4.17992E-01
    6: error= 3.43272E-01
    7: error= 4.65778E-02
  AIN BCE solve succeeded:   7 iterations, error= 4.65778E-02
  Step accepted: perr= 1.35072E+00

BDF2 step     50: T= 3.49457E-01, H= 1.11968E-02, ETAH= 8.35307E-03
    1: error= 1.70092E-01
    2: error= 3.08914E-02
  AIN BCE solve succeeded:   2 iterations, error= 3.08914E-02
  Step accepted: perr= 1.62645E-01

BDF2 step     51: T= 3.60654E-01, H= 2.57669E-02, ETAH= 1.51830E-02
    1: error= 3.52404E-01
    2: error= 1.73789E-01
    3: error= 5.96746E-02
    4: error= 3.03587E-02
  AIN BCE solve succeeded:   4 iterations, error= 3.03587E-02
  Step accepted: perr= 5.94378E-01

BDF2 step     52: T= 3.86421E-01, H= 1.88625E-02, ETAH= 1.32587E-02
    1: error= 1.79701E-01
    2: error= 9.16176E-02
    3: error= 7.43833E-02
    4: error= 1.00017E-01
    5: error= 2.25575E-01
    6: error= 1.09854E-01
    7: error= 3.37595E-02
  AIN BCE solve succeeded:   7 iterations, error= 3.37595E-02
  Step accepted: perr= 5.69380E-01

BDF2 step     53: T= 4.05284E-01, H= 1.79299E-02, ETAH= 1.20551E-02
    1: error= 1.75293E-01
    2: error= 4.70056E-02
  AIN BCE solve succeeded:   2 iterations, error= 4.70056E-02
  Step accepted: perr= 2.10102E-01

BDF2 step     54: T= 4.23213E-01, H= 1.70121E-02, ETAH= 1.14415E-02
    1: error= 1.65401E-01
    2: error= 1.55610E-01
    3: error= 1.07523E-01
    4: error= 1.03868E-01
    5: error= 1.55743E-01
    6: error= 4.79948E-02
  AIN BCE solve succeeded:   6 iterations, error= 4.79948E-02
  Step accepted: perr= 4.54387E-01

BDF2 step     55: T= 4.40225E-01, H= 1.61412E-02, ETAH= 1.08559E-02
    1: error= 1.22362E-01
    2: error= 3.59047E-02
  AIN BCE solve succeeded:   2 iterations, error= 3.59047E-02
  Step accepted: perr= 1.51297E-01

BDF2 step     56: T= 4.56367E-01, H= 1.53150E-02, ETAH= 1.03002E-02
    1: error= 7.49371E-02
    2: error= 2.48126E-02
  AIN BCE solve succeeded:   2 iterations, error= 2.48126E-02
  Step accepted: perr= 8.86794E-02

BDF2 step     57: T= 4.71682E-01, H= 1.45310E-02, ETAH= 9.77293E-03
    1: error= 6.72448E-02
    2: error= 7.18082E-02
    3: error= 5.44777E-02
    4: error= 6.67784E-02
    5: error= 6.12381E-02
    6: error= 3.15914E-03
  AIN BCE solve succeeded:   6 iterations, error= 3.15914E-03
  Step accepted: perr= 1.52167E-01

BDF2 step     58: T= 4.86213E-01, H= 1.37872E-02, ETAH= 9.27267E-03
    1: error= 5.49520E-02
    2: error= 5.45747E-03
  AIN BCE solve succeeded:   2 iterations, error= 5.45747E-03
  Step accepted: perr= 5.85077E-02
